### PR TITLE
Create copy-AutSPACEs-website.md

### PR DESCRIPTION
**Pull request by BrainonSilicon**
Opening this PR based on Issue #636. 

I added the content from the HackMD [here](https://hackmd.io/Sq9w4gwVQKSjmZvoXf8sSQ) (and added linked headings). 

I have 2 questions. 1) was this okay to put in a doc here, and 2) I was thinking that we could use this PR (and maybe an issue) to capture the feedback/discussion that was had in the hackmd. 

What's Needed:
---
- [x] this doc is okay to stay where it is (or not) 
- [x] if I should go ahead and add the discussion about the sections back in. 

I've created this PR as a draft PR so that the discussion can be captured and linked to issues. 

If this is all golden then we can close this specific PR and open an Issue which can discuss editing and finishing the MVP copy.

### CODE REVIEW

**Comment by BrainonSilicon** [comment]
Hey @GeorgiaHCA. 

Just wanted to see if you could review this PR as part of the repo prep ahead of MozFest?

**Comment by BrainonSilicon** [comment]
I am merging this PR as files will be moved as part of the repo refresh

**Comment by GeorgiaHCA** [comment]
Hiya @BrainonSilicon this looks great! Happy to merge

File: platform-designs/design-content/copy-AutSPACEs-website.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+---
+
+# Content for “About Autism” sensory processing, and what the project is 
+
+
+## What is autism?
+
+Autism affects the way people communicate and experience the world around them.
+
+Autism is a spectrum of developmental conditions, including Asperger’s Syndrome. Every autistic person is different. Some are able to learn, live and work independently, while many have learning differences or co-occurring health conditions that require specialist support.
```
Comment (GeorgiaHCA) [inline platform-designs/design-content/copy-AutSPACEs-website.md:19]: Aspberger's syndrome has been folded into ASD in newer DSM versions

File: platform-designs/design-content/copy-AutSPACEs-website.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+You can view your own stories and edit them, or change who they are shared with any time, even once they are submitted.
+
+
+#### Explore AutSPACEs
+
+You do not need an account to explore AutSPACEs and read other people's stories. 
+
+If you are autistic, you may find other people who have similar sense experiences to you. You can discover tips and recommendations from them. 
+
+If you are not autistic, you may find ways to better support n empathise with autistic people by learning from them how their senses affect them.
```
Comment (GeorgiaHCA) [inline platform-designs/design-content/copy-AutSPACEs-website.md:80]: support *and* empathise

**Review by GeorgiaHCA — APPROVED** [review]
Just two minor changes but this is great! Thanks Sophia!

File: platform-designs/design-content/copy-AutSPACEs-website.md (hunk 1 above)
Comment (BrainonSilicon) [reply platform-designs/design-content/copy-AutSPACEs-website.md:19]: On review - I'm wondering if this piece is something we can give a refresh in parts - more reflective of a social model than a medical model - it'll be a tricky one but an important one.

Comment (BrainonSilicon) [reply platform-designs/design-content/copy-AutSPACEs-website.md:19]: For now I've taken out the `including Asperger’s Syndrome.` from line 19.

File: platform-designs/design-content/copy-AutSPACEs-website.md (hunk 2 above)
Comment (BrainonSilicon) [reply platform-designs/design-content/copy-AutSPACEs-website.md:80]: fixed

**Pull request merged**